xbmc/settings/dialogs/GUIDialogSettingsManualBase.cpp

```diff
@@ -239,6 +239,24 @@ CSettingAction* CGUIDialogSettingsManualBase::AddButton(CSettingGroup *group, co
   return setting;
 }
 
+CSettingString* CGUIDialogSettingsManualBase::AddInfoStringButton(CSettingGroup *group, const std::string &id, int label, int level, std::string info,
```

Comment:
Please name the method `AddInfoLabelButton()` to match the `infolabel` format name used in the settings library.